### PR TITLE
Add files to export data required by aimec from com1DFA module

### PR DESCRIPTION
**Pull request by awirb**

### CODE REVIEW

File: avaframe/ana3AIMEC/dfa2Aimec.py [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+# create local logger
+# change log level in calling module to DEBUG to see log messages
+log = logging.getLogger(__name__)
+
+
+def makeAimecDirs(avaDir):
+    """ Make directories where Aimec reads input data from """
+
+    # Set directories for Aimec inputs
+    workDir = os.path.join(avaDir, 'Work/ana3AIMEC/com1DFA')
```
Comment (fso42) [inline avaframe/ana3AIMEC/dfa2Aimec.py:23]: We are going to run into cross plattform problems if we keep using the / in paths. 
So this code should be
os.path.join(avaDir, 'Work', 'ana3AIMEC', 'com1DFA')

File: avaframe/ana3AIMEC/dfa2Aimec.py [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+    # Initialise DEM
+    inputDir = os.path.join(avaDir, 'Inputs')
+    dem = glob.glob(inputDir+os.sep+'*.asc')
+
+    # Load parameters for Aimec postprocessing
+    pressureLimit = float(cfgSetup['pressureLimit'])
+    domainWidth = float(cfgSetup['domainWidth'])
+
+    # Path to com1DFA output in Aimec format
+    workDir = os.path.join(avaDir, 'Work/ana3AIMEC')
```
Comment (fso42) [inline avaframe/ana3AIMEC/dfa2Aimec.py:51]: see above

File: avaframe/ana3AIMEC/dfa2Aimec.py [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+        pfile.write('pathResult=%s,\n' % (os.path.join(workDir, 'AimecResults')))
+
+
+def getDFAData(avaDir, cfgDFA):
+    """ Export the required input data from com1DFA output """
+
+    # Initialise directories
+    inputDir = os.path.join(avaDir, 'Outputs/com1DFA/')
+    workDir = os.path.join(avaDir, 'Work/ana3AIMEC/com1DFA')
+    workDirMain = os.path.join(avaDir, 'Work/ana3AIMEC')
```
Comment (fso42) [inline avaframe/ana3AIMEC/dfa2Aimec.py:80]: join without /

File: avaframe/ana3AIMEC/dfa2Aimec.py [hunk 4]
```diff
@@ -0,0 +1,131 @@
...
+                # log.info('%s%f/%s/raster/%s_%s.asc to the new file %s/dfa_depth/%06d.txt' % (resPath,
+                #                     logDict['Mu'][k], logDict['simName'][k],
+                #                     logDict['simName'][k], logDict['suffix'][m], outputDir, countpfd))
+                countpfd = countpfd + 1
+
+        elif logDict['suffix'][m] == 'ppr':
+            for k in range(sNo):
+                shutil.copy('%s%.03f/%s/raster/%s_%s.asc' % (resPath, logDict['Mu'][k], logDict['simName'][k],
+                                      logDict['simName'][k], logDict['suffix'][m]),
+                                      '%s/dfa_pressure/%06d.txt' % (workDir, countppr))
```
Comment (fso42) [inline avaframe/ana3AIMEC/dfa2Aimec.py:127]: Same / path problem....

File: avaframe/com1DFA/com1DFA.py [hunk 5]
```diff
@@ -50,7 +50,7 @@ def initialiseRun(avaDir, flagEnt, flagRes, inputf='shp'):
     # Set directories for inputs, outputs and current work
     inputDir = os.path.join(avaDir, 'Inputs')
     outputDir = os.path.join(avaDir, 'Outputs/com1DFA')
-    workDir = os.path.join(avaDir, 'Work')
+    workDir = os.path.join(avaDir, 'Work/com1DFA')
```
Comment (fso42) [inline avaframe/com1DFA/com1DFA.py:53]: This and other spots in this file: same / problem

File: avaframe/ana3AIMEC/dfa2Aimec.py (hunk 1 above)
Comment (awirb) [reply avaframe/ana3AIMEC/dfa2Aimec.py]: Adjusted

File: avaframe/ana3AIMEC/dfa2Aimec.py (hunk 3 above)
Comment (awirb) [reply avaframe/ana3AIMEC/dfa2Aimec.py]: done

File: avaframe/ana3AIMEC/dfa2Aimec.py (hunk 4 above)
Comment (awirb) [reply avaframe/ana3AIMEC/dfa2Aimec.py:127]: done

File: avaframe/com1DFA/com1DFA.py (hunk 5 above)
Comment (awirb) [reply avaframe/com1DFA/com1DFA.py]: done

**Comment by awirb** [comment]
I changed all the os.path.join commands to only use quotes instead of /.

**Pull request merged**